meet-data/nzpf/1507/entries.csv

```diff
@@ -10,7 +10,7 @@ Alex Orwin,1992,M,81.9,83,Juniors,130,150,162.5,162.5,100,115,,115,245,-250,-250
 Leethan Tau,1993,M,118.7,120,Juniors,210,-220,220,220,130,140,145,145,230,-240,240,240,605,SBD,Raw,2
 Lucas Mansell,1993,M,136.6,120+,Juniors,220,237.5,250,250,130,137.5,145,145,240,261,272.5,272.5,667.5,SBD,Raw,1
 Simon Kelsall,1970,M,92.1,93,Masters 1,170,177.5,182.5,182.5,112.5,117.5,-125.5,117.5,195,-205,205,205,505,SBD,Raw,1
-Rachel Duggan,1972,F,95.2,84+,Masters 3,110,120,125,125,60,66,70,70,120,132.5,140,140,335,SBD,Raw,1
-Les Laban,1969,M,100.4,105,Masters 5,-220,-220,-220,,115,120,,120,235,252.5,-255,252.5,,SBD,Raw,DQ
+Rachel Duggan,1972,F,95.2,84+,Masters 1,110,120,125,125,60,66,70,70,120,132.5,140,140,335,SBD,Raw,1
+Les Laban,1969,M,100.4,105,Masters 1,-220,-220,-220,,115,120,,120,235,252.5,-255,252.5,,SBD,Raw,DQ
```

Comment:
This one looks like a good catch, although it could also be a typo in the BirthYear.

Reply:
From the results file, this was a mistake when I was adding the meet.